### PR TITLE
Make rviz_common::Panel public

### LINKED ISSUE #260: where is the class rviz_common::Panel?
Hi guys,

I am responsible for migration of rviz from ros1 to ros2. However, I found that the class rviz_common::Panel is not yet available in ros2. Is there any substitute for rviz_common::Panel of ros1? Thanks in advance!

### PR DESCRIPTION
**Pull request by anhosi**
Closes #260.

### CODE REVIEW

**Comment by anhosi** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4419/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1367/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3614/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4484/)

**Comment by anhosi** [comment]
CI:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4454/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1389/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3639/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4524/)

**Comment by anhosi** [comment]
new CI after rebase to master:
* Linux [Build Status](http://ci.ros2.org/job/ci_linux/4470/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/1401/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/3653/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/4536/)

File: rviz_common/src/rviz_common/visualization_frame.hpp [hunk 1]
```diff
@@ -526,6 +526,8 @@ protected Q_SLOTS:
 
   /// Indicates if the toolbar should be visible outside of fullscreen mode.
   bool toolbar_visible_;
+
+  std::string node_name_;
```
Comment (wjwwood) [inline rviz_common/src/rviz_common/visualization_frame.hpp:530]: This seems unrelated. Maybe it was supposed to be part of another pr? I'll not let this postpone merging this, but I'll let you guys follow up if necessary.

**Review by wjwwood — APPROVED** [review]
lgtm

File: rviz_common/src/rviz_common/visualization_frame.hpp (hunk 1 above)
Comment (anhosi) [reply rviz_common/src/rviz_common/visualization_frame.hpp:530]: The `node_name_` member is required for handing the node name over to the PanelFactory constructor that is only called in the `initialize` and not in the VisualizationFrame constructor.

**Pull request merged**